content/index.pug

```diff
@@ -0,0 +1,37 @@
+html
+	head
+		title #{title}
+		meta(name="viewport" content="width=device-width, initial-scale=1")
+		meta(property="og:title" content=metatitle)
+		meta(property="og:url" content=metaurl)
+		meta(property="fb:app_id" content=metaappid)
+		meta(property="og:description" content=metaappdescription)
```

Comment:
can we use a description that fully communicates the meta tag property name? e.g. `meta["property:name"]` perhaps?